config/proto/probe.proto

```diff
@@ -0,0 +1,51 @@
+syntax = "proto3";
+
+package = hermes;
+
+import "github.com/google/cloudprober/metrics/proto/dist.proto";
+
+option go_package = "github.com/googleinterns/step224-2020/config/proto"
+
+message HermesProbeDef {
+  required string probe_name= 1;
+  required targets.TargetsDef targets = 2;
+
+  enum TargetSystem {
+    GCS = 1;
+    CEPH = 2;
+  }
+
+  // Type of Probe
```

Comment:
Can you make the comment a bit more thorough, i.e. Probe types currently supported. Then in the lines right next to to each enum field, you add a one liner explanation for what it does.

Reply:
```suggestion
  // Probe types currently supported
```

Reply:
Also added suggestion below - added comments to all lines of ProbeType.